Added doc for cross org sharing (#58)
mulesoft/docs-access-management: modules/ROOT/pages/external-organization-access.adoc

> = External Organization Access

Reviewer: This title seems a little nonintuitive, despite the fact that the feature we are documenting here is called External Access. 
I wonder if we can use something a bit more "friendly," like "Asset-Sharing Between Organizations" or "Cross-Organizational Collaboration and Asset Sharing" or something really wonderful that I can't find the exact words for just now...?

Author: This is what we decided on with legal for now :/

Author: Worth noting too that this feature is laying ground work for other features and is going to exist independently of the others. Keeping this puppy very isolated.

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> The External Access feature allows separate organizations to collaborate in the Anypoint Platform. For example, organizations can share API specifications, connectors, and other assets in the Anypoint Exchange with each other, without making them public.

Reviewer: allows > enables, in most cases
no "the" before Anypoint Platform or other trademarks or proper nouns, without a common noun following them
```suggestion
The External Access feature enables separate organizations to collaborate in Anypoint Platform. For example, organizations can share API specifications, connectors, and other assets in Anypoint Exchange with each other, without making them public.
```

Administrators control which organizations their users can collaborate with by maintaining a list of trusted organizations in Access Management. Adding an entry to this list means that you trust the other organization and its users, and you acknowledge that you are enabling your users to grant permission to and be granted permissions by the other organization.

> Trust must be established both ways before permissions can be granted. For the users of Organization A to share with the users of Organization B, Organization A must list Organization B as a trusted organization, and Organization B must also list Organization A. If either Organization A or Organization B removes the other, all shared permissions are revoked.

Reviewer: ```suggestion
Trust must be established both ways before permissions can be granted. For the users of Organization A to share with the users of Organization B, Organization A must list Organization B as a trusted organization, and Organization B must likewise list Organization A. If either Organization A or Organization B removes the other, all shared permissions are revoked.
```
Great explanation. I am going to copy and send to Will.

Permissions shared between organizations are granted at the organization level. For example, if Organization A grants Organization B permission to view an asset, it grants that permission to every user in Organization B. Organization A does not receive any information about Organization B's users, sub-organizations, or internal layout.

> 

Reviewer: I really like the previous topic. Great work!

Reviewer: Just FYI, Will refers to "granting organization" and "receiving organization," but that is in the focused context of one-way collaboration rather than both ways. Anyway, I don't know if you guys have compared notes, but you might want to, re. terminology use.

Author: We chatted about it. Granting and receiving is only on the Exchange end of things, so he is okay to use that terminology. We also discussed his changes for two-way, so those should be implemented soon if he hasn't added them already.

== Add a New Trusted Organization

> When you add a new organization to your external access configuration, you establish one half of a trust relationship. After the other organization adds you to its configuration, both organizations are able to share permissions with each other.

Reviewer: ```suggestion
When you add a new organization to your external access configuration, you establish one half of a trust relationship. After the other organization adds yours to its configuration, both organizations can share permissions with each other.
```
"permissions" or "access"? I often see us interchange these, along with other terms, and I always ask myself which is the most precise to this context, and try to use the best one consistently throughout a topic set.

Author: Unfortunately, it's both in this context. They get access to assets elsewhere (dependent on what product is using this framework) but they also get user permissions associated with those assets. It is complicated.

> To add an organization, you must know its Anypoint xref:access-management::organization.adoc#manage-master-organization-settings[domain]. Domains are human-readable names that uniquely identify an Anypoint Platform organization. Ask the administrators of the other organization what theirs is, and be prepared to share your domain with them so they can add your organization to their list.

Reviewer: ```suggestion
To add an organization to your configuration, you must know its Anypoint xref:access-management::organization.adoc#manage-master-organization-settings[domain]. Domains are human-readable names that uniquely identify an Anypoint Platform organization. Ask the administrators of the other organization what their domain is, and be prepared to share your domain with them so they can add your organization to their list in return.
```
Is "human-readable" necessary?

> You can view your organization's unique domain name in the *External Access* page in *Access Management*.

Reviewer: ```suggestion
You can view your organization's unique domain name in the *External Access* page in Access Management.
```

> To add an external organization to your list of trusted organizations:

Reviewer: So, all of this text presupposes an existing list of trusted orgs? What if the audience does not yet have one?

Author: Presumably the org would have other orgs in mind with whom they want to share resources. Their admins talk and swap domain names to each add to their list. As of now, when you add an org to your list, the other org doesn’t get any sort of notification. They have to be in contact.

Author: Presumably the org would have other orgs in mind with whom they want to share resources. Their admins talk and swap domain names to each add to their list. As of now, when you add an org to your list, the other org doesn’t get any sort of notification. They have to be in contact.

Author: Presumably the org would have other orgs in mind with whom they want to share resources. Their admins talk and swap domain names to each add to their list. As of now, when you add an org to your list, the other org doesn’t get any sort of notification. They have to be in contact.

. From the *Access Management* home page, click the *External Access* tab.
. In the *External Access* page, click *Add organization*.
. Enter the unique domain name of the organization you want to share permissions with.
. Click *Add*.

== Remove a Trusted Organization

> When you remove a trusted organization from your external access configuration, both organizations will lose all existing shared permissions. In addition, no new permissions can be granted until a trust relationship is established again.

Reviewer: ```suggestion
When you remove a trusted organization from your external access configuration, both organizations lose all existing shared permissions. In addition, no new permissions can be granted until a trust relationship is reestablished.
```

> To remove a trusted organization from your external access list:

Reviewer: Consider wording in exactly the same way as the Add topic task intro.

> To remove a trusted organization from your external access list:

Author: ```suggestion
To remove a trusted organization from your list of trusted organizations:
```

In turn, an external organization that has your organization's unique domain name can add your organization to its external access list so that your organization can share assets with it. Both organizations must have one another on their external access lists to share resources.

When an external organization elects to share resources with your organization, the external organization does not receive any information about your organization.

You can view your organization's unique domain name in the *External Access* page in *Access Management*.

== Add an Organization as a Trusted Organization

When you add an organization to your list of trusted organizations, that organization can share resources with your organization.
The external organization can revoke your organization's access to its resources anytime.
You should add an organization to your list of trusted organizations only if you trust the external organization.

To add an external organization to your list of trusted organizations:

. From the *Access Management* home page, click the *External Access* tab.
. In the *External Access* page, click *Add organization*.

> . Enter the unique domain name of the organization whose resources you want to view. If you do not know the domain name, contact the organization administrator for that organization.

Author: ```suggestion
. Enter the unique domain name of the organization whose resources you want to access. If you do not know the domain name, contact the organization administrator for that organization.
```
